docs/UserGuide.md

```diff
@@ -141,22 +144,23 @@ Deletes entries of warehouses or suppliers that are not needed anymore.
 
 Format: `delete ct/TYPE i/INDEX`
 
-* The TYPE specified should be one of these values: w / s.
-* The INDEX must be a positive integer, not exceeding the total number of items.
+* The TYPE specified should be one of these values: `w` / `s`.
+* The INDEX must be a positive integer, not exceeding the total length of the displayed list.
 
 Examples:
 
-* `delete ct/w i/1` Removes the warehouse at index 1.
-* `delete ct/s i/12` Removes supplier at index 12 from the list of suppliers.
+* `delete ct/w i/1` Removes the warehouse at index 1 of the list of warehouses.
+* `delete ct/s i/12` Removes supplier at index 12 of the list of suppliers.
 
 ### Deletes a product in a certain warehouse or supplier : `delete`
 
 Deletes a product entry no longer stored by a certain warehouse or sold by a specific supplier.
 
 Format: `delete ct/TYPE i/INDEX pd/PRODUCT_NAME`
 
-* The TYPE specified should be one of these values: pw / ps.
-* The PRODUCT_NAME must be identifiable and starts with alphanumeric character.
+* The TYPE specified should be one of these values: `pw` / `ps`.
+* The INDEX must be a positive integer, not exceeding the total length of the displayed list.
+* The PRODUCT_NAME must be an identifiable full name of the product chosen, and must starts with an alphanumeric character.
```

Comment:
"start".
Also "an identifiable full name of the product" sounds a bit off, perhaps can change to "the"